### DISCUSSION TITLE
0.85.1

### DISCUSSION THREAD

**Discussion by achantavy** [cartography-cncf/cartography#1256] [General]
## What's Changed
* Refactor AWS EKS to new model by @achantavy in https://github.com/lyft/cartography/pull/1247
* Fix: S3 ACL analysis rule for WRITE permissions never triggers by @achantavy in https://github.com/lyft/cartography/pull/1244
* fix: looks like a typo - network interfaces with attached elb wont lo… by @petersutter5 in https://github.com/lyft/cartography/pull/1251
* 0.85.1 by @achantavy in https://github.com/lyft/cartography/pull/1254

## New Contributors
* @petersutter5 made their first contribution in https://github.com/lyft/cartography/pull/1251

**Full Changelog**: https://github.com/lyft/cartography/compare/0.85.0...0.85.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lyft/cartography/releases/tag/0.85.1'>0.85.1</a>.</em>

**Comment by jychp** [comment]
Information are available on [release not](https://github.com/cartography-cncf/cartography/releases/tag/0.85.1).
